lib/cute_pets/pet_fetcher.rb

```diff
@@ -21,6 +21,14 @@ def get_petfinder_pet()
 
     if response.kind_of? Net::HTTPSuccess
       json = JSON.parse(response.body)
+      status_message = json['petfinder']['header']['status']['message']['$t']
+      if status_message == 'shelter opt-out'
+        raise 'The chosen shelter opted out of being accesible via the API'
+      elsif status_message == 'unauthorized key'
+        raise 'Check that your Petfinder API key is configured correctly'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.